module/zfs/dmu_recv.c

```diff
@@ -67,6 +70,7 @@
 int zfs_recv_queue_length = SPA_MAXBLOCKSIZE;
 int zfs_recv_queue_ff = 20;
 int zfs_recv_write_batch_size = 1024 * 1024;
+int zfs_recv_best_effort_corrective = 0;
```

Comment:
`static`

Reply:
think there is a reason none of the tunables are static so that's why the new one isn't static either

Reply:
That reason is purely because #12899 wasn't merged yet. I don't necessarily feel like re-running that on new code when it is.

Reply:
Looks like I would need to wait for #12899 to land before I can make the tunable static. If your PR lands before this patch I'll rebase and add the static keyword.